modules/libpfm/4.11.0.bcr.1/source.json

```diff
@@ -0,0 +1,10 @@
+{
+    "url": "https://netcologne.dl.sourceforge.net/project/perfmon2/libpfm4/libpfm-4.11.0.tar.gz",
+    "integrity": "sha256-XaX4hyveFLNjTJaI2YD2i9ootRAmhyPMEpc+7bq5/sw=",
+    "strip_prefix": "libpfm-4.11.0",
+    "patches": {
+        "module_dot_bazel.patch": "sha256-D7BWrY0WiihCq9XSiVifZWK3sMLbI2chCsgddKJ38kM=",
+        "add_build_file.patch": "sha256-ibpLPyX2v/fbDytUuK4PKeZl/q303bWtSRTU95g5jz0="
+    },
+    "patch_strip": 0
```

Comment:
This should be 1, or change `+++ b/MODULE.bazel` to `+++ MODULE.bazel` in the patch file.

Reply:
Thanks!